api/oc_push.c

```diff
@@ -1854,7 +1857,7 @@ _update_recv_obj(oc_recv_t *recv_obj, const oc_recvs_t *recvs_instance,
   while (rep) {
     switch (rep->type) {
     case OC_REP_STRING:
-      if (strcmp(oc_string(rep->name), "receiveruri") == 0) {
+      if (strcmp(oc_string(rep->name), OC_PUSH_PROP_RECEIVEURI) == 0) {
```

Comment:
why there is no check for:
`if (oc_string_len(rep->name) == OC_CHAR_ARRAY_LEN(OC_PUSH_PROP_RECEIVEURI) &&`